docs/roadmap.md

```diff
@@ -0,0 +1,58 @@
+---
+draft: false
+title: Roadmap
+date: 2023-01-10T09:00:00.000+00:00
+description: Fleek Network's high level roadmap per stage. Devnet, Testnet, and Mainnet.
+category: Documentation
+keywords: [roadmap]
+tags:
+- Roadmap
+- Fleek Network
+---
+
+![Fleek Network: Roadmap](./images/fra.png?202301101924)
```

Comment:
Mind rename the file and provide something a bit more sound, please?

Reply:
Of course! You're right :D